### PR TITLE
Support for statistics callback

### PR DESCRIPTION
**Pull request by etet100**
Basic support for STATISTICS_CALLBACK / setStatsCb callback. Abbility to set `statistics_interval_ms` (which is translated to kafka statistics.interval.ms) is required to make it work.

### CODE REVIEW

File: src/Configuration/Type/StatisticsIntervalMs.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    public function getDefaultValue(): int
+    {
+        return 0;
+    }
+
+    /**
+     * @param mixed $validatedValue
+     */
+    public function cast($validatedValue): int
```
Comment (mariuszjeruzal) [inline src/Configuration/Type/StatisticsIntervalMs.php:54]: AFAIK you'd need to implement CastValueInterface for it to work properly

Comment (etet100) [reply src/Configuration/Type/StatisticsIntervalMs.php]: Do i need it? Its value has to be integer anyway
```
            ->integerNode(StatisticsIntervalMs::NAME)
                ->defaultValue((new StatisticsIntervalMs)->getDefaultValue())
            ->end()
```

Comment (mariuszjeruzal) [reply src/Configuration/Type/StatisticsIntervalMs.php]: Let's take example of LogLevel configuration which implements CastValueInterface

- If you run your consumer like this it will work:
`bin/console kafka:consumers:consume your_consumer` - so values are resolved from sts_gaming_group_kafka.yaml and LogLevel is an integer.

- But when you add this configuration through CLI it is being treated as a string and rdkafka throws exception.
`bin/console kafka:consumers:consume your_consumer --log_level=5`

So basically CastValueInterface is needed for options passed through CLI

Comment (etet100) [reply src/Configuration/Type/StatisticsIntervalMs.php:52]: It makes perfect sense. Fixed.

Comment (mariuszjeruzal) [reply src/Configuration/Type/StatisticsIntervalMs.php:52]: Looks good to me. I think it can be merge but currently I have no write access to this repository. cc @wowo
